charts/ai-software-templates/chatbot/values.schema.json

```
@@ -63,18 +63,56 @@
         },
         "model": {
             "properties": {
+                "existingModelServer": {
+                    "default": false,
+                    "description": "Adds support of existing model server for the deployed application. If selected it overrides the vLLM & llama.cpp use cases.",
```

```suggestion
                    "description": "Adds support of an existing model server for the deployed application. If selected it overrides the vLLM & llama.cpp use cases.",
```

If you address the nit for the readme it will have to be changed here too